docs/common-tasks/inter-vm-communication.md

```diff
@@ -0,0 +1,40 @@
+# Accessing containers from other Virtual Machines
+
+If you use virtual machines to perform cross browser testing you'll need to know
+how to configure VirtualBox networking to allow for communication with your
+containers.
+
+## Networking Mode
+
+In your virtual machine settings, under **Network** choose an enabled adapter
+and then choose NAT for the **Attached to** option.
+
+![VirtualBox Network Settings](../readme_images/virtualbox_network_settings.png)
+
+This will ensure that processes from within virtual machine can route requests
+through the network to your containers.
+
+## DNS
+
+In addition to the network setup, you'll need to choose one of the following
+options so that the DNS entries for your containers can resolve to the proper IP
+address.
+
+### Hosts File
+
+The first option is to manually manage the mapping of container domain names to
+the appropriate IP address. To do this, any time you start or stop containers
+you will need to take the output of `rig dns-records` and update the
+[hosts file](https://en.wikipedia.org/wiki/Hosts_(file)) in the virtual machine.
+
+### DNSDock
```

Comment:
Updated the labelling and dropped DNSDock mentions completely (since they are irrelevant). Also made using DNS the clear preference